alembic/versions/20231124_6af9160a578e_licensepools_time_tracking_flag.py

```diff
@@ -0,0 +1,26 @@
+"""Licensepools time tracking flag
+
+Revision ID: 6af9160a578e
+Revises: 2d72d6876c52
+Create Date: 2023-11-24 08:08:12.636590+00:00
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "6af9160a578e"
+down_revision = "2d72d6876c52"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "licensepools", sa.Column("should_track_playtime", sa.Boolean(), nullable=True)
```

Comment:
Since `null` means the same as `false`, it might be more clear to make this non-nullable, but with a default of `false`, so we don't have to worry about null entries in the future?